### PR TITLE
Prettify JavaScript sandbox logs views

### PR DESCRIPTION
**Pull request by peterp**

### CODE REVIEW

File: go/apps/jsbox/templates/jsbox/jsbox_logs.html [hunk 1]
```diff
@@ -1,23 +1,12 @@
 {% extends "app.html" %}
 {% load url from future %}
 
-{% block extra_header %}
-    <link type="text/css" href="{{STATIC_URL}}css/smoothness/jquery-ui-1.8.11.custom.css" rel="stylesheet" />
-    <link type="text/css" href="{{STATIC_URL}}css/jquery-ui-timepicker.css" rel="stylesheet" />
```
Comment (justinvdm) [inline go/apps/jsbox/templates/jsbox/jsbox_logs.html:6]: Was this an unused dependency, or do we need to add this to `settings.py`?

**Comment by justinvdm** [comment]
Other than the small question, :+1:

File: go/apps/jsbox/templates/jsbox/jsbox_logs.html (hunk 1 above)
Comment (peterp) [inline go/apps/jsbox/templates/jsbox/jsbox_logs.html:6]: Yeah, they're unused and were pretty much included in all the old templates.

**Comment by hodgestar** [comment]
\o/

**Pull request merged**